.changeset/afraid-timers-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-cloud-sdk/openapi-generator': minor
+---
+
+[Improvement] Generated clients support selected top-level media types.
```

Comment:
[suggestion] Currently, the client only supports `application/merge-patch+json`, so change this to say it also additionally supports  `application/merge-patch+json` along with `application/json`